### ISSUE TITLE
use dcat:mediaType URL not blank node

### ISSUE THREAD

**Issue by VladimirAlexiev** [opengeospatial/netCDF-Classic-LD#124]
I think it's better to use `dcat:mediaType https://www.iana.org/assignments/media-types/application/netcdf` rather than
```ttl
 dcat:mediaType [ a dct:MediaType ; dct:identifier "application/netcdf" ] ;
```
But I was unpleasantly surprised that "application/netcdf" is still a provisional type, and doesn't have an individual URL.

---

I've sent this email to: iana@iana.org, cc: Ivan Herman
**individual pages for provisional-standard-media-types?**

Would it be possible to make individual pages for each of the https://www.iana.org/assignments/provisional-standard-media-types/provisional-standard-media-types.xhtml,
like you have for each of the https://www.iana.org/assignments/media-types/media-types.xhtml, eg https://www.iana.org/assignments/media-types/application/gzip ?

The thing is that sem web people want to use individual URLs for media types, eg `dcat:mediaType` demands a URL.
Given the lack of such individual URLs, people are forced to use more complicated representations, eg seehttps://github.com/opengeospatial/netcdf-ld/blob/master/standard/clause_6_informative_text.adoc#worked-example---climate-and-forecasting-conventions :
```ttl
 dcat:mediaType [ a dct:MediaType ; dct:identifier "application/netcdf" ] ;
```

It's true the provisional registry warns:
"This registry is only for temporary use. Entries in this registry are either finalized and moved to the main media types registry or are  abandoned and deleted. Entries in this registry are suitable for use for development and test purposes only."

But consider the case of "application/netcdf". It's been in use since probably 1999, see https://www.unidata.ucar.edu/support/help/MailArchives/netcdf/msg02847.html.
I wouldn't call this "temporary use".

To put it another way, would you recommend that people SHOULD NOT use this URL:
https://www.iana.org/assignments/media-types/application/netcdf

**Comment by VladimirAlexiev** [comment]
I now see that **7.1.2. Requirement A-2** uses this construct:
```
dcat:mediaType [a dct:MediaType;  dct:identifier "application/netcdf"];
dct:format [a dct:MediaType; dct:identifier <http://vocab.nerc.ac.uk/collection/M01/current/NC/>]
```
I suggest to rationalize it to the following, using the fact that `dc:format` is an `rdf:Property` (not `owl:ObjectProperty`):
```
dcat:mediaType <http://vocab.nerc.ac.uk/collection/M01/current/NC/>;
dc:format "application/netcdf";
```
Regarding http://vocab.nerc.ac.uk/collection/M01/current/NC/ :
- Its permanence should be guaranteed (see #128)
- Is it possible to add the triple `dc:format "application/netcdf"` to the semantic payload of that URL?
